scrapers/TheNude.yml

```diff
@@ -0,0 +1,76 @@
+name: TheNude
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.thenude.com/index.php?page=search&action=searchModels&__form_name=navbar-search&m_aka=on&m_name={}
+  scraper: performerSearch
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.thenude.com
+    scraper: performerScraper
+xPathScrapers:
+  performerSearch:
+    performer:
+#      Name: //a[@class="model-name"]/text()
+      Name: //a[@class="model-name"]/../../a/@title
+      URL: //a[@class="model-name"]/@href
+  performerScraper:
+    performer:
+      Name: //meta[@itemprop="name"]/@content

```

Comment:
`(//meta[@itemprop="name"])[1]/@content`
gets rid of the second matched name (usually studio)